share/spice/thesaurus/thesaurus.js

```diff
@@ -6,16 +6,29 @@
             return Spice.failed("thesaurus");
         }
 
+        // Check if user want to find antonym only
+        var searchAntonymOnly = (document.title.indexOf('antonym') > -1);
```

Comment:
`query` variable only contains the word used for the API looking. I changed the code to look up the original query from function `DDG.get_query()`